libs/full/async_cuda/CMakeLists.txt

```diff
@@ -10,10 +10,10 @@ if(NOT HPX_WITH_ASYNC_CUDA)
   return()
 endif()
 
-if(NOT TARGET Cuda::cuda)
+if(NOT TARGET Cuda::cuda AND NOT HPX_WITH_HIP)
   message(
     FATAL_ERROR
-      "Cuda future support has been enabled, but the Cuda installation is not setup"
+      "GPU future support has been enabled, but the GPU installation is not setup"
```

Comment:
Maybe explicitly say that HIP or CUDA hasn't been set up? "the GPU installation" doesn't mean much to a user (or to us).